templates/quarkus/events/.builders.yaml

```diff
@@ -1 +1,3 @@
-default: quay.io/boson/faas-quarkus-builder
+default: quay.io/boson/faas-quarkus-jvm-builder
```

Comment:
@lance note this requires release of buildpacks

Reply:
https://github.com/boson-project/buildpacks/releases/tag/v0.3.0